Make interpolation after crash optional (#113)
precice/micro-manager: docs/configuration.md

</participant>
```

## Interpolate a crashed micro simulation

> If the optional dependency `sklearn' is installed, the Micro Manager can interpolate a crashed micro simulation. To interpolate a crashed micro simulation, set
> `"interpolate_crash": "True"` in the `simulation_params` section of the configuration file.

Reviewer: ```suggestion
If the optional dependency `sklearn' is installed, the Micro Manager will derive the output of a crashed micro simulation by interpolating outputs from similar simulations. To enable this, set
`"interpolate_crash": "True"` in the `simulation_params` section of the configuration file.
```

> For more details on the interpolation see the [interpolation documentation](tooling-micro-manager-running.html/#what-happens-when-a-micro-simulation-crashes).

Reviewer: ```suggestion
For more details on the interpolation see the [crash handling documentation](tooling-micro-manager-running.html/#what-happens-when-a-micro-simulation-crashes).
```
